.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://registry.npmjs.com/
```

Comment:
I think this is redundant as we have the registry information in the package.json